src/App.css

```diff
@@ -0,0 +1,38 @@
+.App {
```

Comment:
Since you used styled components you could probably remove the App.css file

Reply:
There is a gif on the README.md

Reply:
Thanks. I was wondering if I could do that.